website/docs/d/api_management.html.markdown

```diff
@@ -33,7 +33,7 @@ output "api_management_id" {
 
 * `id` - The ID of the API Management Service.
 
-* `additional_location` - One or more `additional_location` blocks as defined below
+* `additional_location` - Zero or more `additional_location` blocks as defined below
```

Comment:
It looks like you may have accidentally added the new resource properties to the data source documentation instead of the resource doc?

Reply:
You're right, I did.  Docs updated.